packages/protocol/migrationsConfig.js

```diff
@@ -51,7 +52,7 @@ const DefaultConfig = {
     targetVotingYieldParameters: {
       initial: 0.00016, // (x + 1) ^ 365 = 1.06
       max: 0.0005, // (x + 1) ^ 365 = 1.20
-      adjustmentFactor: 1 / 365,
+      adjustmentFactor: 1 / 3650,
```

Comment:
In order to start with voter rewards turned off, we'll need to set `initial`, and `adjustmentFactor` here to both be zero. Please preserve the desired values here in a comment that they should be set to those values in order to enable voter rewards.